opentracing-spring-web-starter/src/test/java/io/opentracing/contrib/spring/web/starter/SkipPatternConfigTest.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright 2016-2019 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.spring.web.starter;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+import org.springframework.boot.actuate.autoconfigure.endpoint.web.WebEndpointProperties;
+import org.springframework.boot.actuate.autoconfigure.web.server.ManagementServerProperties;
+import org.springframework.boot.actuate.endpoint.EndpointId;
+import org.springframework.boot.actuate.endpoint.EndpointsSupplier;
+import org.springframework.boot.actuate.endpoint.web.ExposableWebEndpoint;
+import org.springframework.boot.actuate.endpoint.web.WebOperation;
+import org.springframework.boot.autoconfigure.web.ServerProperties;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.assertj.core.api.BDDAssertions.then;
+
+/**
+ * @author Gilles Robert
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+public class SkipPatternConfigTest {
+
+  @Test
+  public void should_pick_skip_pattern_from_web_properties() {
```

Comment:
I think we should be consistent with case. We use camel case in all others tests if I am not mistaken :)

Reply:
Done.